lte/cloud/go/plugin/mconfig_test.go

```diff
@@ -173,6 +174,148 @@ func TestBuilder_Build(t *testing.T) {
+	_ = orc8rplugin.RegisterPluginForTests(t, &plugin.LteOrchestratorPlugin{})
+	builder := &plugin.Builder{}
+
+	nw := configurator.Network{
+		ID: "n1",
+		Configs: map[string]interface{}{
+			lte.CellularNetworkType: models2.NewDefaultTDDNetworkConfig(),
+			orc8r.DnsdNetworkType: &models.NetworkDNSConfig{
+				EnableCaching: swag.Bool(true),
+			},
+		},
+	}
+	gw := configurator.NetworkEntity{
+		Type: orc8r.MagmadGatewayType, Key: "gw1",
+		Associations: []storage.TypeAndKey{
+			{Type: lte.CellularGatewayType, Key: "gw1"},
+		},
+	}
+	lteGW := configurator.NetworkEntity{
+		Type: lte.CellularGatewayType, Key: "gw1",
+		Config: newGatewayConfigNonNat(),
+		Associations: []storage.TypeAndKey{
+			{Type: lte.CellularEnodebType, Key: "enb1"},
+		},
+		ParentAssociations: []storage.TypeAndKey{gw.GetTypeAndKey()},
+	}
+	enb := configurator.NetworkEntity{
+		Type: lte.CellularEnodebType, Key: "enb1",
+		Config:             newDefaultEnodebConfig(),
+		ParentAssociations: []storage.TypeAndKey{lteGW.GetTypeAndKey()},
+	}
+	graph := configurator.EntityGraph{
+		Entities: []configurator.NetworkEntity{enb, lteGW, gw},
+		Edges: []configurator.GraphEdge{
+			{From: gw.GetTypeAndKey(), To: lteGW.GetTypeAndKey()},
+			{From: lteGW.GetTypeAndKey(), To: enb.GetTypeAndKey()},
+		},
+	}
+
+	actual := map[string]proto.Message{}
+	expected := map[string]proto.Message{
+		"enodebd": &mconfig.EnodebD{
+			LogLevel: protos.LogLevel_INFO,
+			Pci:      260,
+			TddConfig: &mconfig.EnodebD_TDDConfig{
+				Earfcndl:               44590,
+				SubframeAssignment:     2,
+				SpecialSubframePattern: 7,
+			},
+			BandwidthMhz:        20,
+			AllowEnodebTransmit: true,
+			Tac:                 1,
+			PlmnidList:          "00101",
+			CsfbRat:             mconfig.EnodebD_CSFBRAT_2G,
+			Arfcn_2G:            nil,
+			EnbConfigsBySerial: map[string]*mconfig.EnodebD_EnodebConfig{
+				"enb1": {
+					Earfcndl:               39150,
+					SubframeAssignment:     2,
+					SpecialSubframePattern: 7,
+					Pci:                    260,
+					TransmitEnabled:        true,
+					DeviceClass:            "Baicells ID TDD/FDD",
+					BandwidthMhz:           20,
+					Tac:                    15000,
+					CellId:                 138777000,
+				},
+			},
+		},
+		"mobilityd": &mconfig.MobilityD{
+			LogLevel: protos.LogLevel_INFO,
+			IpBlock:  "192.168.128.0/24",
+		},
+		"mme": &mconfig.MME{
+			LogLevel:                 protos.LogLevel_INFO,
+			Mcc:                      "001",
+			Mnc:                      "01",
+			Tac:                      1,
+			MmeCode:                  1,
+			MmeGid:                   1,
+			NonEpsServiceControl:     mconfig.MME_NON_EPS_SERVICE_CONTROL_OFF,
+			CsfbMcc:                  "001",
+			CsfbMnc:                  "01",
+			Lac:                      1,
+			RelayEnabled:             false,
+			CloudSubscriberdbEnabled: false,
+			EnableDnsCaching:         true,
+			AttachedEnodebTacs:       []int32{15000},
```

Comment:
you should add the NatEnabled: false here to make the expectation explicit in the test case

Reply:
Test fails if I add false parameter.